<?xml version="1.0"?>

<launch>
  <node name="dead_reckoning_node" pkg="auto_trax_sensors" type="dead_reckoning_node" output="screen">
    <param name="imu_topic" value="imu" />
    <param name="odom_topic" value="odometry" />
  </node>
</launch>
